lib/java/spring/web/src/main/java/co/airy/spring/web/filters/RequestLoggingFilter.java

```diff
@@ -48,6 +58,14 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         } finally {
             if ("POST".equals(requestToUse.getMethod())) {
                 log.info(createMessage(requestToUse));
+                if (analytics != null && requestToUse.getRequestURI().endsWith(".connect")) {
```

Comment:
Not a big fan of having a piece of generic logging middleware in a library depend on business logic. `airy-core/lib/` should be decoupled from everything `airy-core/backend`.

Here's an alternative proposal for still keeping this generic:
1. Create a separate tracking library in `lib/java/spring`
2. Have a component `TrackingRequestFilter` that works like this request logging filter
3. Library users can supply a (list of) bean(s) `RouteTracking` that a) match the request url/method and b) return a `TrackMessage`
4. Inject the `RouteTracking` beans supplied by the library users into the `TrackingRequestFilter`

Thereby you have a generic library that apps can use to track route calls, while keeping the tracking (business) logic separate. `RouteTracking` would be the API contract.

- Names are of course open to change
- There are more ways of designing this than the way I described above. The important part is to separate the "Whenever there is call that matches predicate X do Y" logic from the implementation of X and Y